org.groestlcoin.groestlcoin-qt.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
```

Comment:
Ideally this, the svg icon and the metainfo file should be included in the tarballs you use below.

Reply:
Those tarballs are built by complex gitian scripts similar as https://github.com/flathub/org.bitcoincore.bitcoin-qt/
We rather keep it as it is